pagopa/io-web-profile
# [IOPID-848] : Mixpanel LockProfile Flow

#  ⚠️ Depends on https://github.com/pagopa/io-web-profile/pull/49 ⚠️

## Short description
Adding Mixpanel Track Event for Lock Profile Flow

## How to test
Run the project and check the console for the tracked event (mixpanel mocked), as specified in Figma, for the lock profile flow.
